<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Neue Halle Registrieren - On Padel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ressources/icons/logo_smalltrans.png</normaloff>ressources/icons/logo_smalltrans.png</iconset>
  </property>
  <widget class="QLineEdit" name="halleName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>131</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txtHalleCity">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Adresse*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="halleAddress">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="halleZipCode">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>269</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="halleCity">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="hallePhone">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="txtCoachTelefon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>249</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>PLZ*</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtHalleName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>111</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name*</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtHallePhone">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>311</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Telefon*</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtCoachEmail">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>249</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Stadt*</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtContactPerson">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Ansprechpartner*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contactPersonHalle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>510</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ressources/icons/logo-onpadel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceOneMorning">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>131</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceGroupMorning">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtContactPerson_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>102</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Gruppe</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtContactPerson_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Preise*</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtTitleHalle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Halle Registrieren</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtContactPerson_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Einzel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>691</width>
     <height>671</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ressources/images/bg11.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="txtPriceMorning">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Vormittag</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHalleSave">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(252, 122, 42);
color: rgb(255, 255, 255);
border-radius: 5px;
}
QPushButton:hover{
background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>Registrieren</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceGroupEvening">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtContactPerson_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Preise*</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtOperatingDay">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>283</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Betriebstage*</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="halfframe">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>611</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(156, 139, 116,120);
border-radius: 8px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="txtPriceAfternoon">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Nachmittag</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnMontag">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: rgba(99, 98, 93, 250);
color: rgb(255, 255, 255);
border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Montag</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/icons/plus.png</normaloff>
      <normalon>ressources/icons/check-mark.png</normalon>ressources/icons/plus.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDienstag">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: rgba(99, 98, 93, 250);
color: white
}</string>
    </property>
    <property name="text">
     <string>Dienstag</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/icons/plus.png</normaloff>
      <normalon>ressources/icons/check-mark.png</normalon>ressources/icons/plus.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMittwoch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: rgba(99, 98, 93, 250);
color: rgb(255, 255, 255);
border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Mittwoch</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/icons/plus.png</normaloff>
      <normalon>ressources/icons/check-mark.png</normalon>ressources/icons/plus.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDonnerstag">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: rgba(99, 98, 93, 250);
color: rgb(255, 255, 255);
border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Donnerstag</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/icons/plus.png</normaloff>
      <normalon>ressources/icons/check-mark.png</normalon>ressources/icons/plus.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFreitag">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: rgba(99, 98, 93, 250);
color: rgb(255, 255, 255);
border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Freitag</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/icons/plus.png</normaloff>
      <normalon>ressources/icons/check-mark.png</normalon>ressources/icons/plus.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSamstag">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: rgba(99, 98, 93, 250);
color: rgb(255, 255, 255);
border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Samstag</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/icons/plus.png</normaloff>
      <normalon>ressources/icons/check-mark.png</normalon>ressources/icons/plus.png</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="priceOneEvening">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtPflichtfelder">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Black</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>*Pflichtfelder</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>halfframe</zorder>
  <zorder>txtHalleName</zorder>
  <zorder>txtCoachEmail</zorder>
  <zorder>txtHalleCity</zorder>
  <zorder>txtHallePhone</zorder>
  <zorder>hallePhone</zorder>
  <zorder>halleCity</zorder>
  <zorder>halleName</zorder>
  <zorder>txtCoachTelefon</zorder>
  <zorder>halleAddress</zorder>
  <zorder>halleZipCode</zorder>
  <zorder>txtContactPerson</zorder>
  <zorder>contactPersonHalle</zorder>
  <zorder>Logo</zorder>
  <zorder>priceOneMorning</zorder>
  <zorder>priceGroupMorning</zorder>
  <zorder>txtContactPerson_4</zorder>
  <zorder>txtContactPerson_5</zorder>
  <zorder>txtTitleHalle</zorder>
  <zorder>txtContactPerson_3</zorder>
  <zorder>txtPriceMorning</zorder>
  <zorder>btnHalleSave</zorder>
  <zorder>priceGroupEvening</zorder>
  <zorder>txtContactPerson_6</zorder>
  <zorder>txtOperatingDay</zorder>
  <zorder>txtPriceAfternoon</zorder>
  <zorder>frame</zorder>
  <zorder>priceOneEvening</zorder>
  <zorder>txtPflichtfelder</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
